content/documentation/install-uninstall.md

```diff
@@ -6,116 +6,125 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
 
-    A pop up window may emerge and ask you to type your GitHub username and password, then the download starts.
+5. **PrinceXML** enables you to create a PDF version of your Quire project. At https://www.princexml.com/download/, download either the 32-bit or 64-bit EXE installer depending on your operating system. (If you’re not sure of your system  open the Settings app by pressing Windows+I. Go to System > About and look for the “System type” entry on the right side.) Click on the downloaded file and hit "run", you should see a setup wizard screen that will guide you through install.
 
-    Change directory to the `quire-cli` folder:
+6. **Windows Build Tools** is a set of developer tools for your PC. To install them, open PowerShell as administrator, type the following command, and hit enter:
 
     ```text
-    cd quire-cli
+    npm install --g --production windows-build-tools
     ```
 
-    The following command will install dependencies:
+    This command installs c++ 2015 build tools and python 2 required for node-gyp. The process will take some time and you’ll see the starting prompt with the name of your computer once it’s complete.
 
-    ```tx
-    npm install -g
-    ```
+6. **Quire** is operated through a {{< q-def "command-line interface" >}} (CLI) that enables you to create, preview and output publications using Windows PowerShell with commands like `quire new`, `quire preview` and `quire site`. You can learn more in the [*Quire Commands*](/documentation/quire-cli/) chapter of our guide. Copy and paste the following line into PowerShell to install Quire:
 
-    Installing the dependencies takes some time, and you'll notice multiple text strings being generated on the screen during the process. Wait until the prompt with your username shows up and then, to verify that `quire-cli` has been installed correctly, type:
-
-    ```tx
-    quire -V
+    ```text
+    npm install --global @thegetty/quire-cli
     ```
 
-    If the version number is returned, it means that the install is correct.
+    The Quire installation process may take several minutes, during which time there will be considerable messaging output in PowerShell. The only messages of any concern are those labeled as ERROR or ERR. These likely indicate a failed installation.
 
-7. Lastly, to navigate to your home directory and create a new project or publication type:
+    To verify installation, enter the command below. This will give you a list of [commands](/documentation/quire-cli/) that will help you get started using and navigating Quire. And if you get "command not found" it means it was not installed correctly.
 
-    ```tx
-    cd ~
+    ```text
+    quire --help
     ```
 
+    Search or post to our [Discussions Forum](https://github.com/thegetty/quire/discussions) to troubleshoot installation issues.
+
```

Comment:
Once we have our Get Started page figured out should we maybe link to it here as a next step now that they've installed? Same with Troubleshooting section on Forum.